Classes/Backend/Form/ProfileShowFieldsItemProcFunc.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Fgtclb\AcademicPersons\Backend\Form;
+
+use TYPO3\CMS\Backend\Utility\BackendUtility;
+use TYPO3\CMS\Core\Information\Typo3Version;
+
+class ProfileShowFieldsItemProcFunc
+{
+    /**
+     * @param array<string, mixed> $params
+     *
+     * @return void
+     */
+    public function showFields(&$params): void
+    {
+        // ToDo: Create a registry option to define more fields to select
+        $params['items'] = array_merge($this->defaultFieldItemShowFields(), []);
```

Comment:
Not sure if we should use array_merge here . in the end not needed when combining the result of the default method with an empty array - can be assigned directly.